packages/front/svelte/views/association/association.svelte

```diff
@@ -1,31 +1,43 @@
 <script>
-  import { onMount } from "svelte";
-  import { user as userStore } from "../../store/auth.store";
-  import apiDatasubService from "../../../build/src/shared/apiDatasub.service";
-  import IdentifierHelper from "../../../build/src/shared/helpers/IdentifierHelper";
+  import associationService from "../../services/association.service.js";
+  import Breadcrumb from "../../dsfr/Breadcrumb.svelte";
+  import InfosLegales from "./InfosLegales.svelte";
+  import Tabs from "../../dsfr/Tabs.svelte";
+  import TabContent from "../../dsfr/TabContent.svelte";
 
-  let identifier = document.getElementById("svelte-association").getAttribute("data-identifier");
+  export let route = "";
+  const id = route.split("/")[1];
 
-  let user;
-  let promise;
+  let promise = associationService.getAssociation(id);
 
-  onMount(async () => {
-    user = $userStore;
-    promise = getUserToken();
-  });
-
-  async function getUserToken() {
-    if (IdentifierHelper.isRna(identifier)) apiDatasubService.searchAssoByRna(identifier);
-    else apiDatasubService.searchAssoBySiren(identifier, user);
-  }
+  const tabNames = ["Tableau de bord des subventions", "Pièces administratives", "Établissements"];
+  const segments = [{ label: "Accueil", url: "/" }, { label: `Association (${id})` }];
 </script>
 
-<div>
-  {#await promise}
-    <p>...waiting</p>
-  {:then association}
-    <p>Here is your association : {association}</p>
-  {:catch error}
-    <p style="color: red">{error.message}</p>
-  {/await}
-</div>
+<Breadcrumb {segments} />
+{#await promise}
+  <p>Fetching association...</p>
+{:then association}
+  <InfosLegales {association} />
+  <div class="tab-asso">
+    <Tabs titles={tabNames}>
+      <svelte:fragment slot="tab-content">
+        {#each tabNames as tab, index}
+          {#if index === 0}
+            <TabContent selected={true} {index}>{tab}</TabContent>
```

Comment:
Tu peut pas faire une ternaire genre : 
```suggestion
            <TabContent selected={index === 0} {index}>{tab}</TabContent>
```
Comme ça tu aurais pas besoin du if/else ?

Reply:
Yep